app/src/org/commcare/adapters/EntityListAdapter.java

```diff
@@ -44,11 +45,10 @@
  * @author ctsims
  * @author wspride
  */
-public class EntityListAdapter implements ListAdapter, EntitySortNotificationInterface {
+public class EntityListAdapter extends SortableEntityAdapter implements ListAdapter {
```

Comment:
All of the code deleted from this class was just moved into `SortableEntityAdapter`